If registrations are closed, don't hide the password form field on
the log-in page.

--- a/hc/accounts/views.py
+++ b/hc/accounts/views.py
@@ -57,7 +57,7 @@
         request.profile.save()


-def login(request, show_password=False):
+def login(request, show_password=(not settings.REGISTRATION_OPEN)):
     bad_credentials = False
     if request.method == 'POST':
         form = EmailPasswordForm(request.POST)
